### PR TITLE
Improved properties tutorial

### PR DESCRIPTION
**Pull request by Aggror**
# PR Details
improvements to Properties tutorial for the C# beginners

## Types of changes
- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs [hunk 1]
```diff
@@ -12,40 +13,65 @@ namespace CSharpBeginner.Code
...
+        public CharacterType Character;
+        public enum CharacterType
+        {
+            Warrior,
+            Archer,
+            Mage
+        }
+
+        /// <userdoc> This is a super long description. Use it to help the people that use your components, understand what this propery does.
+        /// The text above is displayed in game studio and also shows up when you hover over the property
```
Comment (Eideren) [inline samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs:58]: Not closing <userdoc> ?

Comment (Aggror) [reply samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs]: Well spotted

File: samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs [hunk 2]
```diff
@@ -12,40 +13,65 @@ namespace CSharpBeginner.Code
...
     {
+        // By default public properties are ordered alphabetically
+        public string SomeString = "Hello Stride";
+        public int SomeInteger = 10;
+
+        // Use DataMember followed by a number to order properties by number
+        [DataMember(5)]
         public bool SomeBoolean = true;
+
+        //Instead of using the name of the variable, you can also define your own text
```
Comment (Basewq) [inline samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs:24]: Space between `//` and text for consistency.

File: samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs [hunk 3]
```diff
@@ -12,40 +13,65 @@ namespace CSharpBeginner.Code
...
         public float SomeFloat = 5.6f;
-        public int SomeInteger = 10;
-        public string SomeString = "Hello world";
-        public Color SomeColor = Color.Red;
-        public Vector2 SomeVector2 = new Vector2(1, 2);
+
+        public Vector2 SomeVector2; // Is the same as = new Vector2(0, 0);
         public Vector3 SomeVector3 = new Vector3(1, 2, 3);
-        public Vector4 SomeVector4 = new Vector4(1, 2, 3, 4);
+        public Vector4 SomeVector4 = new Vector4(5); //All 4 float value get the value of 5
```
Comment (Basewq) [inline samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs:30]: Space between `//` and text for consistency.

File: samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs [hunk 4]
```diff
@@ -12,40 +13,65 @@ namespace CSharpBeginner.Code
...
         public List<string> StringList = new List<string>();
         public List<Entity> EntityList = new List<Entity>();
+        public List<CameraComponent> CameraList = new List<CameraComponent>();
 
         // If we dont want a public property to be visible in the editor we can use '[DataMemberIgnore]'
         [DataMemberIgnore]
-        public string SomeHiddenProperty = "HiddenInEditor";
+        public string HideMe;
+
+        //Enums can be used for dropdowns
```
Comment (Basewq) [inline samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs:48]: Space between `//` and text for consistency.

File: samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs [hunk 5]
```diff
@@ -12,40 +13,65 @@ namespace CSharpBeginner.Code
...
-        // We can reference other entities to our script by using the Entity class
-        public Entity SomeEntity;
+        // Turns a float in to a range slider
+        // Dragging the slider uses the smallstep value
+        // Clicking the slider uses the bigstep value
+        [DataMemberRange(1, 100, 0.1, 1, 3)]
+        public float RangedFloat = 10.0f;
 
-        // If we want a list of ojects like strings, integers or even Entities, we have to create the new List right away
+        // If we want a list of ojects like strings, Entities or specific components, we have to create the new List right away
```
Comment (Basewq) [inline samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/PropertiesDemo.cs:39]: Misspelled 'object' in this comment.

**Pull request merged**